src/test/data/JsonSerializableFinanceTrackerTest/typicalRecordsFinanceTracker.json

```diff
@@ -2,43 +2,43 @@
   "_comment": "FinanceTracker saves file which contains the same Record values as in TypicalRecords#getTypicalFinanceTracker()",
   "records" : [ {
     "name" : "Alice Pauline",
-    "amount" : "120",
+    "amount" : "$120.30",
     "date" : "12/02/2017",
     "description": "",
     "tagged" : [ "friends" ]
   }, {
     "name" : "Benson Meier",
-    "amount" : "119",
+    "amount" : "$119",
```

Comment:
* Value here not stated in 2dp, unlike in `src/test/data/JsonSerializableFinanceTrackerTest/typicalRecordsFinanceTracker.json` where prices listed as absolute values are rounded off to 2dp - should it be standardised to 2dp as well? (hence becomes `$119.00` , similarly for the other test cases in this file too)